docs/source/api_reference/models.rst

```diff
@@ -594,6 +594,12 @@ MXMNet
 .. autoclass:: deepchem.models.torch_models.MXMNet
   :members:
 
```

Comment:
Make sure to add the new layers to the layers.rst. Add unit tests for the layers